platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -139,17 +139,33 @@ event_processor:
   context_path: /
   # The listening port of the event processing server
   port: 8084
+  # The flag to enable the status callback to the client domain
+  client_domain_status_callback_enabled: true
+  # The flag to enable the event delivery to the anchor business server
+  callback_api_event_delivery_enabled: true
   # The queue where the event processor will
   queue:
     type: kafka
     kafka:
       bootstrap_server: localhost:29092
-      event_type_to_topic:
-        all: dev_ap_event_single_queue
-        quote_created: ap_event_quote_created
-        transaction_created: ap_event_transaction_created
-        transaction_status_updated: ap_event_transaction_status_changed
-        transaction_error: ap_event_transaction_error
+
+# The list of clients for the Anchor server to safely communicate with the outside client domains.
+# Each item in the list may contain the following fields:
+#   - domain: the domain of the client
+#   - type: the type of the client. The supported values are `custodial` and `non-custodial`
+#   - public_key: the public key of the client. This must match the public key of the TOML file of the client domain
+#   - url: the URL of the client's callback API endpoint
+#
+# Examples:
+#     - domain: lobstr.com
+#       type: non-custodial
+#       public_key: GBI2IWJGR4UQPBIKPP6WG76X5PHSD2QTEBGIP6AZ3ZXWV46ZUSGNEGN2
+#       url: https://callback.lobstr.com/api/v1/anchor/callback
+#     - domain: mgi.com
+#       type: custodial
+#       public_key: GA22WORKYRXB6AW7XR5GIOAOQUY4KKCENEAI34FN3KJNWHKDZTZSVLTU
+#       url: https://callback.mgi.com/api/v2/anchor/callback
+client_attribution:
```

Comment:
Can the attribution represents some entities other than a `wallet` such as a sending anchor? 

If we have a non-wallet client, the naming will be off.

Reply:
Good point. `clients:`?

Reply:
This is for client attribution purposes. `clients` feels too generic and can be confusing. I don't have better naming though. Can we have another one?

Reply:
I think I disagree, I think including `_attribution` or something that implies that the list of clients is used for a specific purpose might get us into trouble when we find another use case for the list. I'd rather use `clients` for the list and then reference the clients by domain elsewhere in the configuration if necessary for attribution or another use case. wdyt @Ifropc ?

Reply:
I'd prefer it to be `wallets`, but as Jamie said it could be anchors too. So IMO `clients` is better than attribution, not sure if attribution is a good word.

Reply:
Since I don't have better wording either, let's use `clients:`.